docs/en/brand-store.md

```diff
@@ -0,0 +1,198 @@
+          a Brand Store. (Classic systems with no custom model assertion now
+          have a generic one that points at the Ubuntu Store.)
+
+The model assertion contains data such as:
+
+- The Brand Store it points to
+- The model name
+- The Brand Store SSO account-id
+- The gadget snap name (and on Core systems, the kernel snap)
+- And more
+
+As noted, the model assertion is **signed** by a key generated under the Brand
+ Store SSO account and is placed in an image. The image is then ready to go to
+production and distribution. (When you create an image using the `ubuntu-image` 
+tool, the model assertion and various other required assertions are inserted
+ into the image automatically. If you manually create an image, you need to 
+take these steps yourself. See [Seeding a Classic
+ Image][seeding-a-classic-image].)
+
+!!! Note:
+          As mentioned above, it is strongly recommended to use the Brand Store
+          SSO account to generate the key that signs the model assertion. This
+          document assumes you use the Brand Store SSO account for all such
+          purposes.
+
+Using a Brand SSO account key to sign a model used in an image:
+
+![image][brand-store-3]
+
+## Systems pointing at the Brand Store
+
+Here we overview what has been discussed so far:
+
+- Systems contain a model assertion.
+- The model assertion contains the account-id of the Brand SSO account and
+was signed by a key made under that account.
+- The model assertion points at the Brand Store.
+
+!!! Note:
+          This picture still does not show the required system authentication
+          through which a system may gain access to their Brand Store.
+
+Systems whose model points at a Brand Store are rejected unless authentication
+ via a Serial Vault is also used:
+
+![image][brand-store-4]
+
+- Models point systems to their Brand Store
+- Systems can be rejected by the Brand Store as unauthenticated
+
+## Authentication through the Serial Vault
+
+A normally configured Brand Store authenticates a system trying to access it. 
+Authentication is done based on an authentication token called a **macaroon** (see 
+figure below). The macaroon is provided by the Canonical **Authentication
+ Service** after the device has obtained a signed serial assertion. Without a 
+proper serial assertion, the device cannot obtain the macaroon and is rejected 
+by the Brand Store. A proper signed serial assertion is given to a system by
+ the **Serial Vault**. Obtaining a signed serial assertion from the Serial vault
+ normally occurs when the system first boots up.
+
+This process involves a couple of pieces:
+
+- The Serial Vault needs to be configured to support the  model. This setup
+includes the snap account-id of the Brand SSO account, a model name (which
+ must be used in the model assertion mentioned previously), and a key generated
+by the Brand SSO account. Setting up the Serial Vault configuration currently 
+requires communicating with Canonical.
```

Comment:
Let's mention the specifics of how to get in contact with us about this task.

Reply:
not sure. doing this is always under the auspices of a commercial project, so this will be known.

Reply:
See comment above related to this.

Reply:
I'm inclined to update the Contact/Support page first, then see how we handle these afterwards.

Reply:
How about updating the Contact/Support page and then point people to that page in this doc. That'd be good enough for me.